dotcom-rendering/src/web/components/Card/components/CardLayout.tsx

```diff
@@ -26,7 +26,7 @@ const decideDirection = (imagePosition: ImagePositionType) => {
 };
 
 const decideWidth = (minWidthInPixels?: number) => {
-	if (minWidthInPixels) {
+	if (minWidthInPixels !== undefined && minWidthInPixels !== 0) {
```

Comment:
What does it mean to have a negative `min-width`?

```suggestion
	if (minWidthInPixels !== undefined && minWidthInPixels >= 0) {
```